src/vitepress_config.jl

```diff
@@ -138,7 +141,7 @@ function pagelist2str(doc, page::String)
     name = if isnothing(idx)
         splitext(page)[1]
     else
-        elements[idx].text[1]
+        elements[idx].text
```

Comment:
Is there a reason this was deleted?  It's causing the output to be `Any[pagename]` instead of `pagename`, which is not great...

Reply:
```suggestion
        elements[idx].text[1]
```